test/ReportPortal.Shared.Tests/Reporter/ReporterTest.cs

```diff
@@ -659,5 +661,116 @@ public void ShouldThrowWhenExtensionManagerIsNull()
 
             act.Should().Throw<ArgumentNullException>();
         }
+
+        [Fact]
```

Comment:
Better to create separate fixture in `Extensibility\Embedded\` folder.

Reply:
moved.